src/styles/base/_place-holder.scss

```diff
@@ -1,5 +1,5 @@
 .placeholder {
-  --color-place-holder: var(--nypl-colors-ui-gray-light-cool);
+  --color-place-holder: var(--nypl-colors-ui-gray-dark);
```

Comment:
Was this change made for `a11y` reasons?  `gray.dark` is the lightest gray that is compliant on top of white?

Reply:
Yes, it seems like it is the most compliant one _for that font size_. A color matrix would be nice to have (maybe similar to what we had before https://nypl.github.io/design-toolkit/sections/color-a11y.html).